scripts/src/public/services/orders.ts

```diff
@@ -452,7 +448,12 @@ async function orderDbToApi(order: db.Order, userId: string, wallet: string): Pr
 		}, pick(context.meta, "title", "description", "content", "call_to_action"));
 
 		data.title = app.name;
-		data.description = order.isMarketplaceOrder() ? capitalizeFirstLetter(context.type) : "Completed";
+		if (order.isMarketplaceOrder) {
+			const offerContent = (await offerDb.OfferContent.get(order.offerId))!;
```

Comment:
This is good for now, but a better solution would be to include this piece of information inside the order row, so we don't have to query for it on each order. This can be a big performance hit

Reply:
@nitzantomer can you add a ticket for this?

Reply:
@doodyparizada 
https://kinecosystem.atlassian.net/browse/ECO-1034
it's on you